### PR TITLE
Remove support email

### PR DESCRIPTION
**Pull request by wongstein**
This PR clarifies to users in documentation and in error modals to report issues to Github Issues.  

Questions: Remove mailto FC?  A quick search in codebase doesn't show the Mailto FC is used anywhere else.

Also, not tested.

### CODE REVIEW

File: packages/insomnia/src/ui/routes/error.tsx [hunk 1]
```diff
@@ -46,13 +45,7 @@ export const ErrorRoute: FC = () => {
...
       <h1 style={{ color: 'var(--color-font)' }}>Application Error</h1>
       <p style={{ color: 'var(--color-font)' }}>
-        Failed to render. Please send the following error to{' '}
-        <Mailto
-          email="support@insomnia.rest"
-          subject="Error Report"
-          body={errorMessage}
-        />
-        .
+        Failed to render. Please report to Github Issues: https://github.com/Kong/insomnia/issues
```
Comment (jackkav) [inline packages/insomnia/src/ui/routes/error.tsx]: would be good to preserve error messages to share with us in the issue

Comment (jackkav) [reply packages/insomnia/src/ui/routes/error.tsx]: nevermind its shown below.

Comment (gatzjames) [inline packages/insomnia/src/ui/routes/error.tsx]: Nit: We could add a clickable link:
```suggestion
        Failed to render. Please report to Github Issues: <a href="https://github.com/Kong/insomnia/issues">https://github.com/Kong/insomnia/issues</a>
```

File: packages/insomnia/src/ui/components/error-boundary.tsx [hunk 2]
```diff
@@ -65,8 +64,7 @@ class SingleErrorBoundary extends PureComponent<Props, State> {
           // @ts-expect-error -- TSCONVERSION
           message: (
             <p>
-              Failed to render {componentName}. Please send the following error to{' '}
-              <Mailto email="support@insomnia.rest" subject="Error Report" body={error.stack} />.
+              Failed to render {componentName}. Please report the error on our Github Issues: https://github.com/Kong/insomnia/issues
```
Comment (gatzjames) [inline packages/insomnia/src/ui/components/error-boundary.tsx]: Nit: We can add a link to GitHub issues:
```suggestion
              Failed to render {componentName}. Please report the error on our Github Issues: <a href="https://github.com/Kong/insomnia/issues">https://github.com/Kong/insomnia/issues</a>
```

**Review by gatzjames — APPROVED** [review]
👍

**Pull request merged**